modules/nw-ovn-k-anp-nbctl.adoc

```diff
@@ -0,0 +1,499 @@
+name                : a13517855690389298082
+
+_uuid               : ec77552f-f895-4f35-aa52-a4e44cb06e7e
+addresses           : ["172.18.0.3"]
+external_ids        : {direction=Egress, gress-index="1", ip-family=v4, "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:1:v4", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy}
+name                : a10706246167277696183
+
+_uuid               : b406e4a6-4dba-4d03-96b5-383b2c6a44a4
+addresses           : ["10.244.1.3", "10.244.1.4", "10.244.2.3", "10.244.2.4", "10.244.2.5", "10.244.2.6", "10.244.2.7", "10.244.2.8"]
+external_ids        : {direction=Egress, gress-index="2", ip-family=v4, "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:2:v4", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy}
+name                : a18396736153283155648
+
+_uuid               : 42e115fb-d1cb-437b-b3c0-dbf23c46e3b1
+addresses           : ["10.0.54.26/32", "10.0.56.38/32", "10.0.69.31/32", "172.30.0.1/32"]
+external_ids        : {direction=Egress, gress-index="3", ip-family=v4, "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:3:v4", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy}
+name                : a10622494091691694581
+
+_uuid               : 77554661-34c7-4d24-82e2-819c9be98e55
+addresses           : ["10.244.1.3", "10.244.2.7"]
+external_ids        : {direction=Egress, gress-index="4", ip-family=v4, "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:4:v4", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy}
+name                : a5972452606168369118
+
+_uuid               : c8b87fe8-dd6c-4ef3-8cea-e7ea8a5bdb3d
+addresses           : ["0.0.0.0/0"]
+external_ids        : {direction=Egress, gress-index="5", ip-family=v4, "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:5:v4", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy}
+name                : a11452480169090787059
+----
+====
+
+To troubleshoot a specific policy and rule for that ANP, follow these steps:
+
+. Examine the ACL list for that policy by running the following command:
++
+[source,terminal]
+----
+$ ovn-nbctl list acl | grep <policy_name>
```

Comment:
As [commented upstream](https://github.com/ovn-org/ovn-kubernetes/pull/4431#discussion_r1644619505), using `ovn-nbctl find` like in the comment might work better. But it might be wise if we do the same than the upstream docs does, for consistency.